doc/Advanced.md

```diff
@@ -28,6 +32,36 @@
 
 [← Go back to Modules Development](ModulesDevelopment.md) | [↑ Go to the Table of Content ↑](../README.md) | [Continue to Frequently Asked Questions →](FAQ.md)
 
+
+## Plugging the QC to an existing DPL workflow
+
+Your existing DPL workflow can simply be considered a publisher. Therefore, replace `o2-qc-run-producer` with your own workflow. 
+
+For example, if TPC wants to monitor the output of the workflow `o2-qc-run-tpcpid`, modify the config file to point to the correct data and do : 
+```
+o2-qc-run-tpcpid | o2-qc --config json://${QUALITYCONTROL_ROOT}/etc/tpcQCPID.json
+```
+
+## Writing a DPL data producer 
+
+For your convenience, and although this is not a QC problem, we would like to document how to write a simple data producer in the DPL. The DPL documentation can be found [here](https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/README.md) and for questions please head to the [forum](https://alice-talk.web.cern.ch/).
```

Comment:
Maybe "For your convenience, and it does not lie within the QC scope, we would..."? It sounds less harsh.